security/src/main/java/com/fincity/security/controller/ClientController.java

```diff
@@ -145,4 +146,13 @@ public Mono<ResponseEntity<Page<CodeAccess>>> fetchCodes(Pageable pageable,
 		return this.service.fetchCodesBasedOnClient(pageable, clientCode, emailId)
 				.map(ResponseEntity::ok);
 	}
+
+	@GetMapping("/register/events")
+	public Mono<ResponseEntity<Boolean>> evokeRegistrationEvents(ServerHttpRequest request,
+			ServerHttpResponse response,
+			@RequestBody AuthenticationRequest authRequest) {
```

Comment:
User RegistrationRequest not authentication request

Reply:
That will have the password and everything right. Which might have been just used to register.

Reply:
Add userId in that request to get the just created user id.